src/activities/generate-version-json.ts

```diff
@@ -64,8 +42,8 @@ export const generateVersionJson: GenerateVersionJson = async ({ importContentRe
     versionDoi: version.doi,
     article: articleStruct,
     preprintDoi: version.preprint.doi,
-    preprintUrl: version.preprint.content,
-    preprintPosted: version.preprint.publishedDate,
+    preprintUrl: version.preprint.content ?? '',
+    preprintPosted: version.preprint.publishedDate ?? new Date(),
```

Comment:
This feels like it could come back to bite us as a faulty assumption. We should figure out why we don't have a publish date for the preprint I think.